src/x86/rfxcodec_encode_dwt_shift_x86_sse2.asm

```diff
@@ -21,6 +22,29 @@
 
 %ifidn __OUTPUT_FORMAT__,elf
 section .note.GNU-stack noalloc noexec nowrite progbits
+%ifdef PIC
```

Comment:
Can we just require PIC to be defined? That is, if it's not defined, compilation fails.

Also, I've never heard that PIC code cannot be used in statically linked code. It may be sub-optimal, but the we are dealing with large amounts of data here, and the core operation should not be affected.